utilities/tools/Set-ModuleReadMe.ps1

```diff
@@ -627,8 +627,11 @@ Convert the given Bicep parameter block to JSON parameter block
 .PARAMETER BicepParamBlock
 Mandatory. The Bicep parameter block to process
 
+.PARAMETER CurrentFilePath
```

Comment:
Is this change related to the VMSS in particular or can it be integrated into the Set-ModuleReadme PR?

Reply:
This was indeed already in main by now. I merged latest main & resolved the conflict